modules/@apostrophecms/global/index.js

```diff
@@ -42,7 +42,10 @@ module.exports = {
     // Intentionally the same
     pluralLabel: 'apostrophe:globalDocLabel',
     searchable: false,
-    singleton: true,
+    singleton: {
```

Comment:
we should migrate advanced permission module too or ideally support the previous format too. we can have customer using `singleton` as a boolean